src/components/Term/definitions.js

```diff
@@ -179,6 +181,11 @@ export default {
         "複数の演算子を含む式の評価順序を決定するための指標。例えば、乗算演算子は加算演算子より優先順位が高いため、 3 + 4 * 5 は 3 + (4 * 5) と評価される。",
       referencePage: "/docs/trial-session/expressions/",
     },
+    javascriptStringAddition: {
+      name: "文字列の加算",
```

Comment:
[NITS]
```suggestion
      name: "文字列の結合",
```
加算という表現はあるかもしれないが、MDNに合わせた。